Refactor: Improve HTML Structure and Add Application Entry Point

YonLiud opened the pull request:
I've done a few changes that I thought might be contributional

* **HTML Refactoring:**
1) Removed all inline styles from the <body> and <canvas> elements in the index.html file.
2) Improved readability and maintainability by moving styling concerns to external CSS (styles.css).
3) Ensured a cleaner separation of content (HTML) and presentation (CSS), adhering to best practices.

* **Application Entry Point:**
This change enables the application to be run directly as a standalone script.
Like so:
```bash
cd app
python3 __init__.py
```

YonLiud committed:
refactor: clean up HTML by removing inline styles and improving readability

- Removed inline styles from <body> and <canvas> elements to simplify HTML
- Ensured all styling is handled via external CSS for better maintainability
- Preserved content structure and functionality of the page

YonLiud committed:
feat: add entry point for running the application

- Allows the application to be run directly as a standalone script

s1nisteR commented on app/__init__.py:
```diff
@@ -34,4 +32,6 @@ def home():
\ No newline at end of file
+if __name__ == '__main__':
+    app = create_app()
+    app.run()
```
use `debug=True` inside `run()` to make it work similar to the flask command

s1nisteR commented:
Looks good to me. You should additionally modify the `README.md` .

Basically just change `flask --app app run --debug` to `python __init__.py` for now. Also mention that this command launches a development server and should not be used in production.

s1nisteR reviewed (changes requested):
I added a comment on where to make the changes.

YonLiud committed:
misc: add Debug flag to True in app.run

- Under s1nister's request

YonLiud committed:
docs: Update README.md file to latest changes

- Update README to changes made in [0f2931d](https://github.com/omen-osdev/omen-demo/pull/4/commits/0f2931d5e015ecd01d8cb4fdc797964721851d20)

YonLiud commented:
should be good to go by now

s1nisteR commented:
Nvm, I'll modify the README. Everything looks okay. Merging. Nice Work!

YonLiud commented:
no need to modify, already done it in [fbf1d08](https://github.com/omen-osdev/omen-demo/pull/4/commits/fbf1d08cc91e22eb44208a59fc8e9ccad4a6e805)
